fs/expose/dokan/__init__.py

```diff
@@ -165,18 +206,18 @@ def handle_fs_errors(func):
     makes the function return zero instead of None as an indication of
     successful execution.
     """
-    name = func.__name__
     func = convert_fs_errors(func)
+
     @wraps(func)
-    def wrapper(*args,**kwds):
+    def wrapper(*args, **kwds):
         try:
-            res = func(*args,**kwds)
-        except OSError, e:
+            res = func(*args, **kwds)
+        except OSError as e:
             if e.errno:
                 res = -1 * _errno2syserrcode(e.errno)
```

Comment:
Is it right to multiply by -1 here. I think, it scrambles error codes. At least, this may work not the same way in Python as in C/C++.

I tried to open file, which raised PermissionDeniedError in Python, but I did not get any error through Dokan.